### DISCUSSION TITLE
ScriptReference/Events.UnityEvent

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#10] [Notes]
# ScriptReference/Events.UnityEvent

https://docs.unity3d.com/ScriptReference/Events.UnityEvent.html

**Comment by vertxxyz** [comment]
See [`UnityEventTools`](https://docs.unity3d.com/ScriptReference/Events.UnityEventTools.html) for **Editor** tools working with UnityEvents.

**Comment by vertxxyz** [comment]
UnityEvents do not function in edit mode by default. There is a toggle in the inspector, but each event needs to managed.
Doing this via code can be done via [`SetPersistentListenerState`](https://docs.unity3d.com/ScriptReference/Events.UnityEventBase.SetPersistentListenerState.html).
